closes ISS 337.  Export Chart button removed.  Occupation/Salary togg…

jkoren opened the pull request:
…le only on Tree Map visualization.

alexjball reviewed (changes requested):
Please fix the warnings shown under "files changed". You can check that there are no warnings locally by running `npm run lint`. Removing exportPDF might produce new warnings.

alexjball commented on frontend/src/ui/Results/Results.tsx:
```diff
@@ -105,11 +105,53 @@ const Results: React.FC<ResultsProps> = ({
     toggle === 'fill' ? setToggle('opacity') : setToggle('fill');
   };
 
+  const exportChart = '';
```
Let's remove the commented out lines and the `exportChart` constant. We can always pull up this this code again using git.

jkoren commented:
Made requested changes. I removed the commented out lines and the exportChart constant.   
npm run link gave errors on exportPDF, Canvg, and jsPDF which are no longer used, so I removed that code as well.

Errors from lint:
```
warning  'exportPDF' is assigned a value but never used  @typescript-eslint/no-unused-vars
warning  'Canvg' is defined but never used    @typescript-eslint/no-unused-vars
warning  'jsPDF' is defined but never used    @typescript-eslint/no-unused-vars
```
